### DISCUSSION TITLE
How to create A single line black and white line drawing with Foocus

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3285] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/d023c550-4f69-4567-a58e-610bedf3af05)
A single line black
and white line drawing with
no tonal shading of 1 diverse
middle school student doing
a handstand, 1 playing a
small keyboard; students
doing a Yoga stretch; a
naturally lighted classroom

![image](https://github.com/user-attachments/assets/a0d745e2-1093-42e9-a24b-02067cce83c5)
Prompt: A single line black
and white line drawing with
no tonal shading of 3 diverse
middle school students
talking enthusiastically at a
round table, located within a
naturally lighted classroom
with a pendant light

Looking forward to your response. 
Thanks.

**Comment by mashb1t** [comment, marked as answer]
use style pencil drawing or line art (if this matches what you want more)

**Reply by oldhand7** [reply]
Is it possible with Fooocus?

**Reply by mashb1t** [reply]
just select it and try it out
